### DISCUSSION TITLE
Unable to edit the imported records

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#15711] [How to] [answered]
Hi,

I imported around 600 contacts to yetiforce and facing two problems.

1. When i try to edit any fields, i am getting the error as "A Contact with this email address already exists in the system". But if I create any new record and edit it, it works good. While editing, i am facing this error only for the imported contacts. 
![email already exist](https://user-images.githubusercontent.com/96602176/153845571-502a4bed-c85f-4f0f-bb10-cfa3ed7a45fd.PNG)

2. The contact module having one "Related Field 1:M".  The Organization Name field was given to Accounts module. In the details tab, The organization tab was blank. 
![details tab]![details tab](https://user-images.githubusercontent.com/96602176/153845914-833a74b3-1fc6-472b-976a-313203ba7a29.PNG)
But When i try to edit the record, it displays the record. 
![edit fields]![edit fields](https://user-images.githubusercontent.com/96602176/153845956-95728873-1092-4ac0-9d77-cc0cd1a14fee.PNG)

Please guide me to solve the above two issues.

Thanks & regards,
Rajasekaran.

**Comment by rskrzypczak** [comment, marked as answer]
1. Empty the recycle bin
    The duplicate records are most likely located in the bin
2. Empty the recycle bin
    Apparently this organization was also moved to the bin, that's why it can't be viewed in the summary

![image](https://user-images.githubusercontent.com/10221999/153887034-ed89677c-21c4-40b7-861b-4a311a742c9f.png)

**Comment by Ilamparithi-Tamizhanangu** [comment]
Hi,

Thanks for your kind help. It worked and solved my issue. :)

Thanks & Regards,
Rajasekaran.
